components/camel-opentelemetry/src/main/docs/opentelemetry.adoc

```diff
@@ -33,7 +33,7 @@ Dashes need for instances to be encoded for JMS property keys.
 |=======================================================================
 
 
-=== Configuration
+=== Explicit
```

Comment:
I don't think this should be explicit. It's configuration.

Reply:
Thanks. I have changed it to Configuration in https://github.com/apache/camel/pull/7019/commits/be9f710fec4653ddd9b95bbf6869d80f518d0f89 commit.